internal/venice-client-common/src/test/java/com/linkedin/venice/utils/concurrent/CloseableThreadLocalTest.java

```diff
@@ -32,33 +31,29 @@ public void testGet() {
     }
   }
 
-  @Test
-  public void testClose() {
-    CloseableThreadLocal<CloseableClass> numbers = new CloseableThreadLocal<>(() -> new CloseableClass(5));
-    Set<CloseableClass> closeableObjects = new HashSet<>();
+  @Test(timeOut = 30)
```

Comment:
Is 30ms sufficient?

Reply:
It's seconds I guess. We don't do much work in the test. I can bump it to 60 sec if you think 30 sec to low.

Reply:
The unit is ms. 30s should be enough - but we'd need `30 * Time.MS_PER_SEC`:

```java
/**
   * The maximum number of milliseconds this test should take.
   * If it hasn't returned after this time, it will be marked as a FAIL.
   */
  public long timeOut() default 0;
```

Reply:
Oh my bad. Thanks for the clarification. I did test this with 10,000 invocations/30ms timeout and it didn't fail. But to be on the safe side, let me bump it up.